src/utils/index.js

```diff
@@ -11,6 +9,15 @@ export const apiInstance = axios.create({
   }
 });
 
+export const validateToken = (token) => {
+  let isValid = false;
+  if (token) {
+    const decoded = jwtDecode(token);
+    isValid = decoded.exp > Date.now() / 1000 && decoded.exp && decoded;
```

Comment:
You have this hierarchy wrong. Should be more like:
```suggestion
    isValid = decoded && decoded.exp && decoded.exp > Date.now() / 1000;
```
You should first check for the object before you check for a property within the object

Reply:
True. Would change this.